src/pages/taeYeong/Detail/Detail.scss

```
@@ -0,0 +1,213 @@
+.detail {
+  box-sizing: border-box;
+
+  margin: 0;
+
+  ul,
+  p,
+  h1,
+  h2,
+  h3 {
+    margin: 0;
+    padding: 0;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+    color: black;
+  }
+
+  .container {
+    width: 960px;
+    margin: 0 auto;
+    /* border: 1px solid black; */
+  }
+
+  .fas.fa-heart {
+    color: red;
+  }
+
+  #navbar {
+    height: 45px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #f4f4f2;
+    margin-bottom: 16px;
+  }
+
+  .navbar__logo {
+    font-family: 'Dancing Script', cursive;
+    font-size: 20px;
+    font-weight: bold;
+    margin-left: 20px;
+  }
+
+  .navbar__menu {
+    display: flex;
+  }
+
+  .navbar__menu li {
+    margin-right: 20px;
+  }
+
+  .navbar__menu a {
+    color: #b9b9b2;
+    font-size: 13px;
+    font-weight: bold;
+  }
+
+  /* detail */
+
+  .detail-title {
+    margin-bottom: 15px;
+  }
+
+  .detail-page__content__title {
+    border-bottom: 3px solid black;
+  }
+
+  .detail__route {
+    text-align: end;
+    font-size: 12px;
+    color: #9c9c9c;
+  }
+
+  .detail-page {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .detail-page__content {
+    width: 530px;
+  }
+
+  .detail-page__content-title {
+    position: relative;
+    padding-bottom: 15px;
+    border-bottom: 2px solid black;
+  }
+
+  #detail-page__content-heart.far.fa-heart {
+    position: absolute;
+    right: 10px;
+    top: 20px;
+    color: #cecece;
+    font-size: 20px;
+  }
+
+  #detail-page__content-heart.fas.fa-heart {
+    position: absolute;
+    right: 10px;
+    top: 20px;
+    color: red;
+    font-size: 20px;
+  }
+
+  .detail-page__content-desc {
+    padding: 15px 0;
+  }
+
+  .nutrition-info h1 {
```

밑에 하단에도 비슷한 클래스들이 많이 있는 것 같은데 세부적으로 네스팅으로 묶어주면 뭔가 가독성 측면에서도 더 높아지지 않을까 생각합니다. 현업에서는 그렇게 scss를 맞추는 버릇을 잘 해줘야 좋다고 저도 수정해야 하지만 태영님도 하시면 좋을 것 같습니다 ㅎㅎ
```suggestion
  .nutrition-info__list {
    display: flex;
    justify-content: space-between;
    margin-bottom: 20px;
                 li {
                    display: flex;
                    justify-content: space-between;
        }
}
```

---

네 맞습니다 ㅎㅎ sass를 사용하지 않아도 별 문제가 없어 그냥 보여주기 식으로 맨 바깥에만 네스팅을 했습니다 ㅋㅋㅋ... 귀찮더라도 sass 문법으로 다 바꾸어 보겠습니다!